pkg/applier/applier.go

```diff
@@ -93,22 +93,19 @@ type Destroyer interface {
 	Errors() status.MultiError
 }
 
-// Dormammu is an incredibly powerful being that dwells in the Dark Dimension,
-// known as the Destroyer of Worlds. He can apply and destroy whole sets of
-// resource objects at will.
+// Supervisor is a bulk client for applying and deleting a mutable set of
+// resource objects. Managed objects are tracked in a ResourceGroup inventory
+// object.
 //
-// Not to be confused with The Doctor, who was named the Destroyer of Worlds by
-// Davros, creator of the Daleks.
-//
-// Yes, the name is silly. But applier.ApplierDestroyer is repetitive and
-// doesn't pass lint checks.
-type Dormammu interface {
+// Supervisor satisfies both the Applier and Destroyer interfaces, with a shared
+// lock, preventing Apply and Destroy from running concurrently.
+type Supervisor interface {
 	Applier
 	Destroyer
 }
 
-// applier is the default implimentation of the ApplierDestroyer interface.
-type applier struct {
+// supervisor is the default implimentation of the Supervisor interface.
+type supervisor struct {
 	// inventory policy for the applier.
```

Comment:
I guess the comments on each field also need to be updated to supervisor.
The `applyMux` prevents both concurrent Apply() calls and Destroy() calls, maybe rename to `actionMux`?
The `errs` field is cleared at the start of the `Applier.Apply` and `Destroyer.Destroy` methods.

Reply:
Pushed a new commit with consistency fixes